features/dd-sdk-android-flags/src/main/kotlin/com/datadog/android/flags/featureflags/FlagsClient.kt

```diff
@@ -320,9 +320,12 @@ interface FlagsClient {
                     NoOpFlagsRepository()
                 }
 
-                val flagsNetworkManager = DefaultFlagsNetworkManager(
+                val callFactory = sdkCore.createOkHttpCallFactory {}
```

Comment:
minor: maybe it is worth to use default lambda argument in the `createOkHttpCallFactory` signature? Otherwise such call with empty lambda looks a bit strange.